.changeset/healthy-boxes-poke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/db': minor
+---
+
+Adds support for self-hosted LibSQL
```

Comment:
I'll come back to this after we know what's going in docs, but for a new feature released in a minor, we typically would hype up the feature a bit here! Something covering the PR description like:

``````suggestion
Adds Astro DB support for a remote LibSQL server instance using any supported LibSQL protocols. This enables Astro DB to be used with self-hosting and air-gapped deployments.

To configure, set `ASTRO_STUDIO_REMOTE_DB_URL`.... (code block example showing where this is set*)
``````
(We'd also need to note whether this is a Studio feature or DB feature)

Reply:
> We'd also need to note whether this is a Studio feature or DB feature

It is a DB feature, not a Studio feature. For environments that can't use Studio, it can be replaced with a LibSQL server.

> To configure, set `ASTRO_STUDIO_REMOTE_DB_URL`.... (code block example showing where this is set*)

There is no code block for this. No code needs to be changed at all, just the environment variables.

Reply:
OK, then we should show setting/changing this as an environment variable e.g. in a `.env` file? Does the environment variable need to be named `ASTRO_STUDIO_REMOTE_DB_URL` ?

It would be nice to show someone what they need to do to take advantage of this, I think! If you'd prefer to make the docs PR first, so that we have that to pull from to write this changeset, that's fine too!

Reply:
I confirm as the blog writer that an example would be neat.

Reply:
Updated my suggestion! What would you like to see different here @Fryuni ?

Reply:
LGTM, @Princesseuh do you think the changeset and docs are good for the blog post or would you prefer to have a project deployed with it to have a link as an example?

Reply:
I'll take a bigger look tomorrow, but this looks fine to me!